Makefile

```diff
@@ -38,3 +46,12 @@ build-server: swig
 
 clean: clean-sdk
 	rm -rf bin gen/gen_wrap.cxx gen/gen.go
+
+appimage: build-module
```

Comment:
Are you planning to do a separate PR to create the github workflow that will make the appimage (like this one: https://github.com/viamrobotics/slam/blob/main/.github/workflows/appimage.yml)? Sorry I missed this in the first round of review.

Reply:
Yes! I'll add you as a reviewer.

Reply:
thanks!